<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintDialog</class>
 <widget class="QWidget" name="PrintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:checked
{
    background-color: #3daee9;
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton_print">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Imprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quitte">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remarque: le dernier objet enregistré est imprimé!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_showDebitPatient">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Débit Patient</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_showTopDeJour">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Top de Jour</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_showScan3D">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Scan 3D</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_showProfile">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Profile</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_showPicDeBragg">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Pic de Bragg</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_showSOBP">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>SOBP</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
